src/Fmt_ast.ml

```diff
@@ -948,8 +948,13 @@ and fmt_pattern c ?pro ?parens ({ctx= ctx0; ast= pat} as xpat) =
       let parens =
         parens || Poly.(c.conf.parens_tuple_patterns = `Always)
       in
+      let wrap =
```

Comment:
It would probably be better to use a longer name that doesn't shadow `Fmt.wrap`.